docs/extensions/index.md

```diff
@@ -1,5 +1,15 @@
 # Extensions
 
+An OCDS release or record package may declare one or more extensions. Extensions can add to the schema, add new codelists, or add entries to existing codelists. 
+
+* **Core extensions** are maintained as part of the standard governance process.
+* A directory of **community extensions** is maintained to allow users to discover common properties and data models they may wish to use for additional data;
+* Publishers may declare their own **local extensions** to suppress validation warnings about additional fields or codelist values in their data, and to document additional fields in their data. 
+
+In addition, a collection of extensions may be packaged together as an **OCDS profile**. When used as part of a profile, extensions may additionally: remove scheme elements and codelist entries. 
+
+The structure of an extension is documented via the [standard extension template](https://github.com/open-contracting/standard_extension_template/blob/master/README.md).
```

Comment:
To avoid duplication of content, I think we should simply refer to http://standard.open-contracting.org/latest/en/extensions/ to define core and community extensions and to http://standard.open-contracting.org/latest/en/extensions/developing/ to develop extensions.

See https://github.com/open-contracting/standard/issues/634